src/odemis/acq/fastem.py

```diff
@@ -167,6 +167,75 @@ def __init__(self, name, coordinates):
         # if None -> acquire, if not None, don't acquire again
 
 
+SCANNER_CONFIG_OVERVIEW = {
+    "multibeam_mode": False,
+    "external": False,
+    "blanker": None,  # automatic
+    "immersion": False,  # disable to get a larger field of view
+    "horizontal_fov": 1.5e-3,
+    "resolution": (1024, 884),  # px
+}
+
+
+SCANNER_CONFIG_MEGAFIELD = {
+    "multibeam_mode": True,
+    "external": False,
+    "blanker": False,
+    "immersion": True,
+    "horizontal_fov": 2.2e-5,
+    "resolution": (1024, 884),
+}
+
+
+SCANNER_CONFIG_LIVESTREAM = {
+    "multibeam_mode": False,
+    "external": False,
+    "blanker": None,  # automatic
+    "immersion": True,
+    "horizontal_fov": 2.2e-5,
+    "resolution": (1024, 884),
+}
+
+
+def _configure_scanner(scanner, multibeam_mode, external, blanker, immersion, horizontal_fov, resolution):
+    """
+    Configure the scanner by setting the VAs in the right order.
+    :param scanner (model.Emitter):
+    :param multibeam_mode (bool):
+    :param external (bool):
+    :param blanker (bool or None): None for automatic blanking at the end of the acquisition or when pausing the stream
+    :param immersion (bool):
+    :param horizontal_fov (float):
+    :param resolution (int, int):
+    """
+
+    scanner.multiBeamMode.value = multibeam_mode
+    scanner.external.value = external
+    scanner.blanker.value = blanker
+    # Immersion needs to be set before changing the horizontalFoV, as the range is updated
+    scanner.immersion.value = immersion
+
+    scanner.horizontalFoV.value = horizontal_fov  # m
+
+    # => compute the scale needed in X, use the same one in Y, and then compute the Y resolution.
+    # => set resolution to full FoV, and then adjust
+    scale = scanner.shape[0] / resolution[0]
+    scanner.scale.value = (scale, scale)
+    if scanner.resolution.value != TILE_RES:
+        logging.warning("Unexpected resolution %s on e-beam scanner, expected %s",
+                        scanner.resolution.value, TILE_RES)
+
+    md = scanner.getMetadata()
+    if model.MD_SINGLE_BEAM_ROTATION in md:
+        scanner.rotation.value = md[model.MD_SINGLE_BEAM_ROTATION]
+    else:
+        scanner.rotation.value = SINGLE_BEAM_ROTATION_DEFAULT
+        logging.warning("Scanner doesn't have SINGLE_BEAM_ROTATION metadata, using %s rad.",
+                        scanner.rotation.value)
+    md[model.MD_ROTATION_COR] = -scanner.rotation.value
```

Comment:
This was only needed for overview image acquisition. If I remember correctly in order to display the overview image at the same angle as the megafield image (where we do not use MD_ROTATION_COR) in Odemis. So you would need to distinguish here between live/overview and megafield as well.
@pieleric Can you comment or confirm, so we can add a comment here for clarity?
Is this the MD that goes on the image?